paris/src/test/java/de/fraunhofer/ids/messaging/paris/ParisServiceTest.java

```diff
@@ -0,0 +1,224 @@
+
+
+    @Configuration
+    static class TestContextConfiguration {
+        @MockBean
+        private KeyStoreManager keyStoreManager;
+
+        @MockBean
+        private ConfigurationModel configurationModel;
+
+        @MockBean
+        private ConfigContainer configurationContainer;
+
+        @MockBean
+        private DapsTokenProvider dapsTokenProvider;
+
+        @MockBean
+        private DapsPublicKeyProvider dapsPublicKeyProvider;
+
+        @MockBean
+        private DapsValidator dapsValidator;
+
+        @MockBean
+        private IdsHttpService idsHttpService;
+
+        @MockBean
+        private MessageService messageService;
+
+        @MockBean
+        private Connector connector;
+
+        @Bean
+        public Serializer getSerializer() {
+            return new Serializer();
+        }
+
+        @Bean
+        public ParisService getBrokerService() {
+            return new ParisService(configurationContainer, dapsTokenProvider, messageService);
+        }
+    }
+
+
+    @BeforeEach
+    public void setUp() throws Exception{
+        this.fakeToken = new DynamicAttributeTokenBuilder()
+                ._tokenFormat_(TokenFormat.JWT)
+                ._tokenValue_("fake Token")
+                .build();
+        final var endpoint = new ConnectorEndpointBuilder()
+                ._accessURL_(URI.create("https://isst.fraunhofer.de/ids/dc967f79-643d-4780-9e8e-3ca4a75ba6a5"))
+                .build();
+        connector = new BaseConnectorBuilder()
+                ._securityProfile_(SecurityProfile.BASE_SECURITY_PROFILE)
+                ._outboundModelVersion_("4.0.0")
+                ._inboundModelVersion_(Util.asList("4.0.0"))
+                ._curator_(URI.create("https://isst.fraunhofer.de/ids/dc967f79-643d-4780-9e8e-3ca4a75ba6a5"))
+                ._maintainer_(URI.create("https://isst.fraunhofer.de/ids/dc967f79-643d-4780-9e8e-3ca4a75ba6a5"))
+                ._hasDefaultEndpoint_(endpoint)
+                .build();
+        Mockito.when(configurationContainer.getConnector()).thenReturn(connector);
+        Mockito.when(configurationContainer.getConfigurationModel()).thenReturn(configurationModel);
+        Mockito.when(configurationModel.getConnectorDeployMode()).thenReturn(ConnectorDeployMode.TEST_DEPLOYMENT);
+        Mockito.when(dapsTokenProvider.provideDapsToken()).thenReturn("Mocked Token.");
+        Mockito.when(dapsPublicKeyProvider.providePublicKeys()).thenReturn(null);
+        Mockito.when(dapsValidator.checkDat(fakeToken)).thenReturn(true);
+        Mockito.when(dapsValidator.checkDat(fakeToken)).thenReturn(true);
+        Mockito.when(dapsTokenProvider.getDAT()).thenReturn(fakeToken);
+        this.notificationMessage = new MessageProcessedNotificationMessageBuilder()
+                ._issued_(IdsMessageUtils.getGregorianNow())
+                ._issuerConnector_(
+                        URI.create("https://w3id.org/idsa/autogen/baseConnector/691b3a17-0e91-4a5a-9d9a-5627772222e9"))
```

Comment:
Having the same URI for all objects can very quickly cause critical bugs! I don't think it is that crucial in this test, but please generally make sure to use distinct URIs. If you, for example, use the same URI twice for objects you describe, then after parsing them, both objects will end up with the properties of the other object, too.